plugins/field-date/test/field_date_test.mocha.js

```diff
@@ -45,6 +44,12 @@ suite('FieldDate', function() {
     assertFieldValue(field, run.value);
   };
 
+  // TODO(https://github.com/google/blockly/issues/3903): Re-enable test cases
+  //  after fixing.
+  invalidValueRuns[3].skip = true;
```

Comment:
Why set skip here instead of in the definition of the test cases? This is counting on the index of the cases staying the same, which seems unstable.

Reply:
In this test suite I want skip `true` before the construction tests and skip `false` after (before `setValue` tests).
I think using the index for setting true and for false also makes it easier to make sure the setting the skip flag `true`/`false` line up (possibly making it more obvious if there is an error with index). Also, the skip true right before the call to run the creation tests seems a little clearer for what the skip is being set for.

I agree using index feels a bit unstable, but I feel like its not too much overhead to update the index as its not expected for the testCases to be expanded frequently, if at all.